### PR TITLE
[New Topic] Rust

### PR DESCRIPTION
**Pull request by KTom101**
### Description

Homepage for new topic Rust.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/rust/rust.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Rust'
+Description: 'Rust is a programming language first released in 2015. With its focus on type safety, concurrency, and performance, it is a safe and practical language to write low level code while avoiding some of the drawbacks and security vulnerabilities associated with other languages like C and C++.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/rust'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+Rust is a programming language first released in 2015. With its focus on type safety, concurrency, and performance, it is a safe and practical language to write low-level code while avoiding some of the drawbacks and security vulnerabilities associated with other languages like [C](https://www.codecademy.com/resources/docs/c) and [C++](https://www.codecademy.com/resources/docs/cpp).
```
Comment (caupolicandiaz) [inline content/rust/rust.md:9]:
```suggestion
**Rust** is a programming language first released in 2015. With its focus on type safety, concurrency, and performance, it is a safe and practical language to write low-level code while avoiding some of the drawbacks and security vulnerabilities associated with other languages like [C](https://www.codecademy.com/resources/docs/c) and [C++](https://www.codecademy.com/resources/docs/cpp).
```

File: content/rust/rust.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Rust'
+Description: 'Rust is a programming language first released in 2015. With its focus on type safety, concurrency, and performance, it is a safe and practical language to write low level code while avoiding some of the drawbacks and security vulnerabilities associated with other languages like C and C++.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/rust'
```
Comment (caupolicandiaz) [inline content/rust/rust.md:4]: I don't believed we have an catalog content for rust yet

**Review by caupolicandiaz — COMMENTED** [review]
@KTom101 Just a couple of notes, otherwise looks good

File: content/rust/rust.md (hunk 2 above)
Comment (yangc95) [inline content/rust/rust.md:4]:
```suggestion
```

**Review by yangc95 — APPROVED** [review]
LGTM!

File: content/rust/rust.md (hunk 2 above)
Comment (yangc95) [reply content/rust/rust.md]: @caupolicandiaz I think we can omit this for now

**Comment by KTom101** [comment]
@yangc95 Thanks! 🎉

**Pull request merged**